experiences/top_articles/top_articles.test.js

```diff
@@ -0,0 +1,89 @@
+const { bootstrapExperience } = require("../../e2e/utils/bootstrap");
+const { scrollTo } = require("../../e2e/utils/scroll");
+const { touchCard } = require("../../e2e/utils/touch");
+const { isAtSnapPoint } = require("../../e2e/utils/snapPoints");
+const { isCardContentLoaded } = require("../../e2e/utils/cardContent");
+const { expect } = require("chai");
+const { getUrlFixture } = require("../../e2e/utils/fixtureUrl");
+const experience = require("./top_articles.json");
+
+describe("top_articles experience", function () {
```

Comment:
we have a linter in this repo now, try`npm run lint` to check for linting issues